webserver/Models/Company.cs

```diff
@@ -1,11 +1,24 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
 namespace webserver.Models;
-
 public class Company : BZEAccount {
 
-    public Company() {
+    public Company(){
+        RealEstates = new List<RealEstate>();
+    }
+
+    public Company(string id, string username, string email, string phonenumber){
+        RealEstates = new List<RealEstate>();
+        Id = id; UserName = username; Email = email; PhoneNumber = phonenumber;
```

Comment:
Please, do not save lines of code.
This doesn't help understanding the code.
Split each of these statements in a single line

```suggestion
        Id = id;
        UserName = username;
        Email = email;
        PhoneNumber = phonenumber;
```